Makefile

```diff
@@ -188,7 +192,7 @@ help:
 	@echo '  BASE_IMAGE - Do not compute the base image to be used as container base from BASEOS_REPO'
 	@echo '               and BASEOS_TAG. Instead, use the base image specified. The BASEOS_ vars will'
 	@echo '               still be used to determine the ceph-container source files to use.'
-	@echo '               e.g., BASE_IMAGE="myrepo/mycustomubuntu:mytag"'
+	@echo '               e.g., BASE_IMAGE="mycustomubuntu:mytag"'
 	@echo ''
```

Comment:
I think it would be useful to add an example here showing how the `BASEOS_REGISTRY` and `BASE_IMAGE` work to define a full base image path similar to what exists on line `e.g., TAG_REGISTRY="myreg" DAEMON_TAG="mydaemontag" will tag daemon "myreg/mydaemontag"`
`